Namespace List notes (#1683)
sensu/sensu-docs: content/sensu-go/5.12/reference/rbac.md

sensuctl namespace list
{{< /highlight >}}

> _Note: If logged in with a non Admin user, `sensuctl namespace list` will list only the available namespaces that the user has access to._

Reviewer: ```suggestion
_Note: If logged in with a non-admin user, `sensuctl namespace list` will list only the available namespaces that the user has access to._
```

I see other areas of the docs site use this format. Or we can get rid of it entirely and say something like "If logged with with a user that does not have administrative access, ..."

> _Note: If logged in with a non-admin user, `sensuctl namespace list` will list only the available namespaces that the user has access to._

Author: ```suggestion
_NOTE: If logged in with a non-admin user, `sensuctl namespace list` will list only the available namespaces that the user has access to._
```

> _NOTE: If logged in with a non-admin user, `sensuctl namespace list` will list only the available namespaces that the user has access to._

Reviewer 2: ```suggestion
_NOTE: For licensed tier users,`sensuctl namespace list` will list only the namespaces that the current user has access to._
```

## Role bindings and cluster role bindings

> A **role binding** assigns a **role** or **cluster role** to users and groups within a namespace.

Reviewer 2: Good catch! 👍
